opta/core/kubernetes.py

```diff
@@ -217,6 +222,82 @@ def current_image_tag(layer: "Layer",) -> Optional[str]:
     return None
 
 
+def create_namespace(layer_name: str) -> None:
+    load_kube_config()
+    v1 = CoreV1Api()
+    namespaces = v1.list_namespace(field_selector=f"metadata.name={layer_name}")
+    if len(namespaces.items) == 0:
+        v1.create_namespace(
+            body=V1Namespace(
+                metadata=V1ObjectMeta(
+                    name=layer_name, annotations={"linkerd.io/inject": "enabled"}
+                )
+            )
+        )
+
+
+def create_manual_secrets(layer_name: str) -> None:
```

Comment:
Should be named `create_manual_secrets_if_not_exists`

Reply:
done